gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/converter/DistcpConverter.java

```diff
@@ -85,7 +85,7 @@ public String convertSchema(String inputSchema, WorkUnitState workUnit) throws S
     modifyExtensionAtDestination(fileAwareInputStream.getFile());
     try {
       InputStream newInputStream = inputStreamTransformation().apply(fileAwareInputStream.getInputStream());
```

Comment:
I think these files should be non-splittable i.e. they likely will not benefit from block-level splitting.

Reply:
I think you might want to make that explicit that SPLIT_ENABLED will work correctly only in case of IdentityConverters. Unless, each block is gzipped/encrypted separately.